### DISCUSSION TITLE
how to create own animated bmp

### DISCUSSION THREAD

**Discussion by keshin88** [pixelit-project/PixelIt#192] [Q&A] [answered]
hello again sorry for the questions, i am unable to understand or work out how to create my own animated bmp. i make different frames in the pixel creator but i cant work out how to make them all into a animated one,

**Comment by o0shojo0o** [comment, marked as answer]
Simply put together as an array, e.g.:
Image 1 = [0,14783,25023,...]
Image 2 = [0,0,0,...]
Image 3 = [0,0,34192,...]
Image 4 = [0,0,13094,...]

animated = [[0,14783,25023,...],[0,0,0,...],[0,0,34192,...],[0,0,13094,...]]

Note that is not unbergrentz many farmes go (so about 6 - 8) because otherwise the memory of the ESP overflows.
